driver-adapters/planetscale-cf-basic/src/index.js

```diff
@@ -0,0 +1,170 @@
+import { Prisma, PrismaClient } from '@prisma/client/edge'
+import { Client } from '@planetscale/database'
+import { PrismaPlanetScale } from '@prisma/adapter-planetscale'
+
+export default {
+  async fetch(request, env, ctx) {
+    const client = new Client({
+      url: env.DRIVER_ADAPTERS_PLANETSCALE_CF_BASIC_DATABASE_URL,
+      // taken from planetscale's docs
+      fetch(url, init) {
+        delete init["cache"];
+        return fetch(url, init);
```

Comment:
This is weird, what does it do? Where is the link to the docs?

Reply:
No explanation, really

Reply:
I started a discussion with PlanetScale as it is also in their official example:
https://prisma-company.slack.com/archives/C01CUJDJ7UN/p1700563614090689
Was added here: https://github.com/planetscale/cloudflare-workers-quickstart/blame/4e7ad2ee0a4b088a6df497c0c3e065e34ef44c77/worker/src/index.js#L10C29-L10C51